nerfstudio/cameras/camera_paths.py

```diff
@@ -155,6 +157,8 @@ def get_path_from_json(camera_path: Dict[str, Any]) -> Cameras:
             camera_type == CameraType.EQUIRECTANGULAR
             or camera_type == CameraType.OMNIDIRECTIONALSTEREO_L
             or camera_type == CameraType.OMNIDIRECTIONALSTEREO_R
+            or camera_type == CameraType.VR180_L
+            or camera_type == CameraType.VR180_R
```

Comment:
Can you condense this to
`if camera_type in [CameraType.EQUIRECTANGULAR, ...]

Reply:
Sounds good, I have updated the formatting.